scripts/filter_expected_warnings.py

```diff
@@ -79,26 +93,27 @@ def filter_expected_warnings(expected_warnings_path):
         ignore_too_many_nodes(sys.stdin.readlines()))
 
     # print unexpected warnings
-    for warning in new_warnings:
-        if not warning.is_in_list(expected_warnings):
-            print(warning.firstline)
-            for line in warning.otherlines:
-                print('  ' + line)
+    unexpected_warning_found = list1_is_in_list2(
+        new_warnings, expected_warnings, '')
```

Comment:
`'+'`?

Reply:
I considered that, but I think it's better to report warnings in the same way as normal in the case where new problems have been introduced and old ones haven't been fixed.